[IMP] Documents: share section to be improved (#9288)
odoo/documentation: content/applications/productivity/documents.rst

possible to access the related :guilabel:`Journal Entry` or add :guilabel:`Tags`.

.. note::
   - The :guilabel:`Contact` is a person related to the document, who has only viewing access rights
     to the document. I.e.: an existing supplier in your database.
   - The person who creates a document is by default :guilabel:`Owner` and has full access rights to
     it. It is possible to replace the owner of a document by selecting someone else from the

>      dropdown list of the :guilabel:`Owner` field.

Author: Thank you for the suggestion, it makes sense to me, I will keep it in mind :)

.. tip::

>    An employee must be owner of a document to be able to see it in **My Profile**.

Author: Feedback from FRGI: employees must be users to see a document.

:guilabel:`Note`, if needed. Then, click :guilabel:`Request`. A placeholder for the missing document
is created in the workspace.

When your document is available, click the placeholder to upload it in its stead. By default, you
are set as the document :guilabel:`Owner`, and the requested person is set as the

> :guilabel:`Contact`.

Author: The requested person can indeed upload the document; however, I double-checked, and they do not become the Contact person for the document.

> :guilabel:`Contact`.

Author: I tried to simplify the sentence.

:guilabel:`Name` your share link, set a validity date by filling in the :guilabel:`Valid Until`
field, tick the :guilabel:`Include Sub Folders` box if you want to share the workspace's
sub-folders, and if you own more than one site, select the :guilabel:`Website` you
want so the share link reflects the right domain name.

Then, allow users to either :guilabel:`Download` files from your workspace, or to :ref:`Download and
Upload <documents/upload>` files to it.

.. Note::
   - The links added to your workspace using the :ref:`Add a Link <documents/add-a-link>` option
     cannot be shared and are, therefore, excluded.
   - When tags are applied to a shared workspace, users can exclusively access the documents
     associated with those tags.

.. _documents/upload:

Upload document
^^^^^^^^^^^^^^^

Select the :guilabel:`Download and Upload` option to enable users to upload their files to your
workspace.

To allow the upload of documents by email, you can create an email alias. The domain name is set by
default but can be modified by clicking on it. The documents sent to this email alias are uploaded

> using the chosen tags to the workspace.

Author: I absolutely agree. Updated accordingly.

By default, the :guilabel:`Document Owner` is the person who uploads a file to a workspace, but you
can select someone else. You can also set a :guilabel:`Contact`, usually an external person, such as
a partner.

> You can check the :guilabel:`Create a new activity` box to automatically create an activity when a

Reviewer: ```suggestion
Enable :guilabel:`Create a new activity` to automatically create an activity when a
```
for the sake of simplifying the sentence
